packages/grafana-data/src/transformations/fieldReducer.test.ts

```diff
@@ -63,6 +63,16 @@ describe('Stats Calculators', () => {
     expect(stats.count).toEqual(2);
   });
 
+  it('should handle undefined field data without crashing', () => {
+    // eslint-ignore @typescript-eslint/no-explicit-any
```

Comment:
@grafana/grafana-frontend-platform I'm not sure why this eslint-ignore isn't working / preventing betterer from ignoring this (`any` is necessary here)

Reply:
Odd. Is it disabled for test files for some reason?